Add 3 documentation rules to Comelit (#37956)
home-assistant/home-assistant.github.io: source/_integrations/comelit.markdown

{% include integrations/config_flow.md %}

{% configuration_basic %}
  host:
    description: The IP address of the Comelit SmartHome device.
  port:
    description: The TCP port of the Comelit SmartHome device.
  pin:
    description: The PIN of the Comelit SmartHome device.
  type:
    description: The type the Comelit SmartHome device.
    keys:
      bridge:
        description: Comelit Serial Bridge.
      vedo:
        description: Comelit VEDO System.
{% endconfiguration_basic %}

> ## Removing the integration
> 
> This integration follows standard integration removal. No extra steps are required.
> 
> {% include integrations/remove_device_service.md %}

Reviewer: ```suggestion
```

## Data updates

This integration fetches data from the device every 5 seconds by default.

## Alarm control panel

> The integration will create an alarm entity for each area. Additionally, it will create a sensor and a presence detection binary sensor for each zone, enhancing monitoring capabilities.

Reviewer: ```suggestion
The integration will create an alarm entity for each area. Additionally, it will create a sensor and a presence detection binary sensor for each zone, enhancing monitoring capabilities.

## Removing the integration

This integration follows standard integration removal. No extra steps are required.

{% include integrations/remove_device_service.md %}

```

> 

Reviewer: maybe outside the scope of this PR, but is there anything I need to have or have done before installing the integration? do I need to have set up the device, set a pin, have an app, a login and account or anything like that, for example?

Author: Yeah, the device needs to be setup; pin is optional, there are no login or account.

{% include integrations/config_flow.md %}

{% configuration_basic %}
  host:
    description: The IP address of the Comelit SmartHome device.
  port:

>     description: The TCP port of the Comelit SmartHome device.

Reviewer: Is there a default port?

Author: Yes, standard HTTP port 80

>     description: The TCP port of the Comelit SmartHome device.

Reviewer: ```suggestion
    description: The TCP port of the Comelit SmartHome device. The default is port 80 (standard for HTTP).
```

  pin:
    description: The PIN of the Comelit SmartHome device.
  type:

>     description: The type the Comelit SmartHome device.

Reviewer: ```suggestion
    description: The type of Comelit SmartHome device.
```

    keys:
      bridge:
        description: Comelit Serial Bridge.
      vedo:
        description: Comelit VEDO System.
{% endconfiguration_basic %}

## Data updates

> This integration fetches data from the device every 5 seconds by default.

Reviewer: ```suggestion
This integration {% term polling polls %} data from the device every 5 seconds by default.
```
Replacing fetch by poll because we have a glossary entry for it :-D

{% include integrations/config_flow.md %}

> {% configuration %}

Reviewer: ```suggestion
{% configuration_basic %}
```
For the integration with UI configuration, we usually use the configuration_basic block.

  host:

>     required: true

Reviewer: ```suggestion
```

    description: The IP address of the Comelit SmartHome device.

>     type: string

Reviewer: ```suggestion
```

  port:

>     required: true

Reviewer: ```suggestion
```

    description: The TCP port of the Comelit SmartHome device.

>     type: string

Reviewer: ```suggestion
```

  pin:

>     required: false

Reviewer: ```suggestion
```

    description: The PIN of the Comelit SmartHome device.

>     type: integer

Reviewer: ```suggestion
```

  type:

>     required: true

Reviewer: ```suggestion
```

    description: The type the Comelit SmartHome device.

>     type: list

Reviewer: ```suggestion
```

    keys:
      bridge:
        description: Comelit Serial Bridge.
      vedo:
        description: Comelit VEDO System.

> {% endconfiguration %}

Reviewer: ```suggestion
{% endconfiguration_basic %}
```
